src/src/WebServer/HTML_wrappers.cpp

```diff
@@ -310,9 +310,8 @@ void html_add_script(bool defer) {
   addHtml(F("<script"));
 
   if (defer) {
-    addHtml(F(" defer"));
+    addHtml(F(" defer>"));
   }
```

Comment:
You should have the `addHtml('>');` after the `if (defer) { .... }` part
Otherwise there is no closing `>` for `defer == false`